modules/installation/proc-install-rhdh-osd-gcp-operator.adoc

```diff
@@ -17,6 +17,48 @@ You can install {product-short} on {osd-short} on {gcp-short} using the {product
 . In the *Filter by keyword* box, enter {product-short} and click the *{product} Operator* card.
 . On the *{product} Operator* page, click *Install*.
 . In the {ocp-short} console, navigate to *Installed Operators* and select *{product} Operator*.
+. Create a secret named `{my-product-secrets}` and add a key named `BACKEND_SECRET` with a `Base64-encoded` string as value:
++
+--
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: {product}
+      baseUrl: https://<rhdh_dns_name>
+    backend:
+      auth:
+        externalAccess:
+            - type: legacy
+              options:
+                subject: legacy-default-config
+                secret: "${BACKEND_SECRET}"
+      baseUrl: https://<rhdh_dns_name>
+      cors:
+        origin: https://<rhdh_dns_name>
+----
+--
+
+. Create a secret named `{my-product-secrets}` and add a key named `BACKEND_SECRET` with a `Base64-encoded` string as value:
++
+--
+[source,yaml,subs="+attributes,+quotes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {my-product-secrets}
+stringData:
+  # TODO: See https://backstage.io/docs/auth/service-to-service-auth/#setup
+  BACKEND_SECRET: "xxx"
+----
+--
+
```

Comment:
For consistency, consider that these steps should be exactly the same as the steps found in `modules/configuring/proc-provisioning-your-custom-configuration.adoc`.